templates/strings/structured-swift4.stencil

```diff
@@ -6,6 +6,7 @@ import Foundation
 
 // swiftlint:disable superfluous_disable_command
 // swiftlint:disable file_length
+// swiftlint:disable function_parameter_count
```

Comment:
I'd add this a bit lower, on line 41, with the other disables around the `enum`. I'd also re-enable it again after the `enum`. (same feedback for the other templates)